root/layout.tt

```diff
@@ -130,12 +130,11 @@
             </p>
 
             <p class="right">
-                [% l('Cover Art provided by the {caa|Cover Art Archive}. Hosted by {host|Digital West}. Sponsored by: {url1|Google},
-                      {url2|OSUOSL} and {more|others...}. ',
-                    { host => 'https://www.digitalwest.com/',
-                      url1 => 'https://www.google.com/',
-                      url2 => '//osuosl.org/',
-                      more => 'https://metabrainz.org/sponsors',
+                [% l('Brought to you by {MeB|MetaBrainz Foundation} and our {spon|sponsors} and {supp|supporters}. 
+                      Cover Art provided by the {caa|Cover Art Archive}.',
```

Comment:
This string is actually slightly different from the one in the .js file, since it has extra spaces after `{supp|supporters}.`. We'll need to put it on a single line like the .js file has so xgettext sees them the same.